GVRf/Framework/framework/src/main/java/org/gearvrf/x3d/Vertex.java

```diff
@@ -0,0 +1,22 @@
+package org.gearvrf.x3d;
+
+public class Vertex {
+	public float[] point = { 0, 0, 0 };
+
+	
+	public Vertex (float x, float y, float z) {
+		this.point[0] = x;
+		this.point[1] = y;
+		this.point[2] = z;
```

Comment:
Appreciate this.  Eventually, we want to do away with Vertex.java and similar programs (VertexNormals.java, textureCoordinates.java, coordinates.java) used solely to create these array lists.  We're in a rush to get the first launch out there prior to an optimization stage.  The java programs are a holdover from when the project began in February, based on a publicly released X3D parser.